server/graphql/resolvers.ts

```diff
@@ -151,24 +151,32 @@ const resolvers = {
     },
     createClient: (_, { client }, { dataSources }): Promise<ServerResponseInterface> => dataSources.clients.create(client),
     updateClient: (_, { client }, { dataSources }): Promise<ServerResponseInterface> => dataSources.clients.update(client),
-    softDeleteClient: (_, { id }, { dataSources}): Promise<ServerResponseInterface> => dataSources.clients.softDelete(id)
+    softDeleteClient: (_, { id }, { dataSources }): Promise<ServerResponseInterface> => dataSources.clients.softDelete(id)
   },
   RequestGroup: {
-    numOpen: (parent, __, { dataSources }): Number => 
-      parent.requestTypes.map(id => {
+    numOpen: (parent, __, { dataSources }): Number => {
+      const openRequestsInRequestTypes = parent.requestTypes.map(id => {
         return filterOpenRequests(getRequestsById(dataSources.requestTypes.getById(Types.ObjectId(id)).requests, dataSources)).length
-      }).reduce((total, num) => total + num, 0),
```

Comment:
Don't see why reduce here would be causing an error https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce